packages/ui/src/views/chatmessage/ChatMessage.jsx

```diff
@@ -664,6 +676,11 @@ export const ChatMessage = ({ open, chatflowid, isAgentCanvas, isDialog, preview
                 return allMessages
             })
         }
+
+        if (data.followUpPrompts) {
```

Comment:
When you refresh the page, follow up prompt is gone.

At Line 939, we need to also display the follow up prompt during initial useEffect. First check if chatflow has followup prompt status is true, and check if chatmessage has follow up prompt, if both fulfiled, display

Reply:
Updated